lumispy/signals/common_luminescence.py

```diff
@@ -107,6 +109,10 @@ def scale_by_exposure(self, exposure=float("nan"), inplace=False):
         -----
         Sets `metadata.Signal.scaled` to `True`. If intensity units is 'counts',
         replaces them by 'counts/s'.
+
+        .. deprecated:: 0.2
+          The `exposure` argument was renamed `integration_time`, and its use
+          will be deprecated in LumiSpy 1.0.
```

Comment:
```suggestion
          The `exposure` argument was renamed `integration_time`, and it will
          be removed in LumiSpy 1.0.
```
The deprecation starts now, as we discourage to use it but it is still possible to use until it is removed in 1.0.